src/Authenticator/FormAuthenticator.php

```diff
@@ -12,7 +12,8 @@
  */
 namespace Authentication\Authenticator;
 
-use Cake\Routing\Router;
+use Cake\Http\ServerRequest;
```

Comment:
Looks like `cakephp/cakephp` dependency is required for this class to be loaded.

I'm still not convinced that this should rely on HTTP and/or Routing packages. Could we just support string urls as @markstory suggested? `loginUrl` option could accept an explicit url string (like `/login`), an array of urls (`['/login', '/users/login']`) or a regex (`%^/[a-z]+/login$%`).

Reply:
@robertpustulka would you mind doing the change then? I've explicitly asked before I started on the change but didn't get any clear yes or no on this... So please change it as you like until it fits what you want.

Reply:
@burzum Sorry, I may have not been clear on this here: https://github.com/cakephp/authentication/pull/146#issuecomment-346813351

I’ll take a look on this tomorrow.